scripts/zones/Dragons_Aery/mobs/Nidhogg.lua

```diff
@@ -26,7 +26,7 @@ function onMobFight(mob, target)
     end
 
     if battletime >= twohourTime then
-        mob:useMobAbility(956)
+        mob:useMobAbility(949)
```

Comment:
Can set this to be the ID we discovered in the Nidhogg PR~!

Reply:
This is now pretty much a merge conflict with the other PR, should I change this here?

Reply:
can pull your other one into here after it merges and resolve the conflict, or rebase this onto the other one. If you haven't had much practice with rebase, this is a good opportunity (back up your scripts 1st if not confident)

Reply:
I just merged release into this, I'll keep avoiding rebase as long as possible T_T